permissions/plugin-console-navigation.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "console-navigation"
+github: "jenkinsci/console-navigation-plugin"
+paths:
+  - "org/jenkins-ci/plugins/console-navigation"
```

Comment:
This should be `io/jenkins/plugins/console-navigation` to match your groupId in your pom.xml (https://github.com/jenkinsci/console-navigation-plugin/blob/develop/pom.xml#L12)